app/services/parameter_safe_render.rb

```diff
@@ -26,14 +49,19 @@ def render_object(object)
     when Hash
       object.merge(object) { |k, v| render_object v }
     else
-      # Don't know how to parse this, send it back
       object
     end
   end
 
+  def renderer_scope
+    @renderer_scope ||= Foreman::Renderer.get_scope(klass: Foreman::Renderer::Scope::Partition, host: host)
```

Comment:
I would neee to check, but doesn't the scope holds e.g. template name? Does this work for rendering snippets correctly? Don't they have different variables also?

Reply:
I'm wondering if `Partition` is the right scope?

Reply:
Good questions, wrong place :-)

If I am touching this line, I will only memorize this for now.